docs/UserGuide.md

```diff
@@ -262,6 +262,10 @@ Exits the program.
 
 Format: `exit`
 
+### Changing the theme of the app
+
+Click Theme to toggle between Dark Theme and Light Theme.
```

Comment:
Maybe we can add screenshots here to show that dark/light themes look like?

Reply:
I agree

Reply:
good idea!